Converters/Civil3d/Speckle.Converters.Civil3dShared/ToSpeckle/Properties/PartDataExtractor.cs

```diff
@@ -16,12 +16,31 @@ public PartDataExtractor() { }
   /// <returns></returns>
   public Dictionary<string, object?>? GetPartData(CDB.Entity entity)
   {
-    if (entity is CDB.Part part)
+    return entity switch
     {
-      return ParsePartData(part.PartData);
+      CDB.Part part => ParsePartData(part.PartData),
+      CDB.PressurePart pressurePart => ParsePartData(pressurePart.PartData),
+      _ => null
+    };
+  }
+
+  private Dictionary<string, object?> ParsePartData(CDB.PressureNetworkPartData partData)
+  {
+    var result = new Dictionary<string, object?>();
+    foreach (CDB.PressurePartProperty prop in partData)
+    {
+      if (!prop.HasValue)
+      {
+        continue; // don't send null props
+      }
+
+      if (!result.ContainsKey(prop.DisplayName))
+      {
+        result.Add(prop.DisplayName, prop.Value);
```

Comment:
for `CDB.Part`, we create a complex dictionary with value, name, context, and units etc. For `CDB.PressurePart`, we simply store the value directly without "metadata" .. intentional? if so, ignore this :)

Reply:
Intentional, apparently pressurepart data is a lot simpler and doesn't contain units or context eg

Reply:
Gotcha. I'm happy!